ppr-ui/src/components/tables/RegistrationTable.vue

```diff
@@ -3,8 +3,9 @@
     <date-picker
       v-show="showDatePicker"
       ref="datePicker"
-      :setEndDate="submittedEndDate"
+      :setEndDate=null
```

Comment:
Not to introduce more work, but maybe to clean up tech debt while at it: `setEndDate` and `setStartDate` are passing a _Date_ type object, but `endDate` and `startDate` are _String_ type props in `DatePicker`.

Reply:
Dima is 100% right here. 
Also, you shouldn't need to set `null` explicitly? Whats the default prop value on the component that it requires null here?

Reply:
Also, does this impact PPR registrations? I know this table is a hybrid between the two.

Reply:
I made the above change and changed the type in the datepicker. I tested both ppr and mhr and any other components that use the date picker and they all work.